packages/blade/src/components/ButtonGroup/StyledButtonGroup.tsx

```diff
@@ -10,22 +10,19 @@ const StyledButtonGroup = styled(BaseBox)<StyledButtonGroupProps>(
     return {
       display: 'flex',
       width: isFullWidth ? '100%' : 'fit-content',
-      overflow: 'hidden',
```

Comment:
was this causing issue with focus ring? (storybook is broken right now so unable to preview before and after https://razorpay.slack.com/archives/G01B3LQ9H0W/p1710930648131329)

Reply:
Yes.